content/200-orm/200-prisma-client/000-setup-and-configuration/050-databases-connections/index.mdx

```diff
@@ -265,6 +265,22 @@ If you consistently experience connection pool timeouts after configuring the re
 
 ### Increasing the pool size
 
+:::
```

Comment:
Why do we add these info sections into these unrelated doc pages? 🤔

Reply:
I thought it's related: this section is about setting the connection pool size, which today is done via a URL param on the connection string but when using QC you need to configure that manually based on the driver you use, no?